lighttpd-ipv6.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Enable IPV6 if needed
+if [[ "${IPV6_DISABLE}" == "1" ]]; then
```

Comment:
My bad, it should be better with
```suggestion
if [[ "${IPV6_DISABLE}" != "1" ]]; then
```